### PR TITLE
Update UI text of 'Continuous Deployments'

### PR DESCRIPTION
**Pull request by Patrick-LuoYu**
Signed-off-by: Patrick-LuoYu patrickluo@yunify.com

## What type of PR is this?
/kind documentation

## What this PR does / why we need it:
Update UI text of 'Continuous Deployments'.

## Special notes for reviewers:
/assign @harrisonliu5 @LinuxSuRen 

## Does this PR introduced a user-facing change?
```release-note
None
```

### CODE REVIEW

File: locales/en/l10n-devopsProjects-codeRepo-list.js [hunk 1]
```diff
@@ -16,14 +16,12 @@
...
  */
 module.exports = {
-  CODE_REPO_PL: 'Code Repository',
-  CODE_REPO_LOW: 'Code Repository',
+  CODE_REPO_PL: 'Code Repositories',
+  CODE_REPO_LOW: 'code cepository',
   CODE_REPO: 'Code Repository',
-  CODE_REPO_DESC:
-    'Code repositories are used to pull code repositories based on GIT or SVN management.',
+  CODE_REPO_DESC: 'Manage code repositories based on Git or SVN.',
```
Comment (LinuxSuRen) [inline locales/en/l10n-devopsProjects-codeRepo-list.js:22]: We don't support SVN here.

File: locales/en/l10n-devopsProjects-codeRepo-list.js [hunk 2]
```diff
@@ -16,14 +16,12 @@
...
   CODE_REPO_EMPTY_DESC:
-    'Import a code repository, you can select a code repository as the code source and choose GitHub, GitLab, Bitbucket, Git and SVN as code source',
+    'Import a code repository from GitHub, GitLab, or Bitbucket. You can also use Git or SVN to directly pull source code.',
```
Comment (LinuxSuRen) [inline locales/en/l10n-devopsProjects-codeRepo-list.js:24]: We don't support SVN here.

**Review by LinuxSuRen — CHANGES_REQUESTED** [review]
Missing something?

image

**Comment by Patrick-LuoYu** [comment]
Fixed

**Comment by Patrick-LuoYu** [comment]
@LinuxSuRen @harrisonliu5 Please take a look. Thanks.

**Comment by LinuxSuRen** [comment]
> Fixed

I need to review all the files again considering you push the commit changes with the flag `--force`. I highly recommend don't use the flag `--force` often. If you'd like to know more recommendations about it, please feel free to read [here](https://github.com/LinuxSuRen/open-source-best-practice/blob/master/how-to-contribute.md).

**Comment by Patrick-LuoYu** [comment]
> > Fixed
> 
> I need to review all the files again considering you push the commit changes with the flag `--force`. I highly recommend don't use the flag `--force` often. If you'd like to know more recommendations about it, please feel free to read [here](https://github.com/LinuxSuRen/open-source-best-practice/blob/master/how-to-contribute.md).
> 
> I'm doing the test with the following image:
> 
> ```
> surenpi/ks-console:latest@sha256:1c6e0e0f3a89b4b7ad74f63c5a2472fb3fac3dd9d167068b8f0c4e3f9cdb334e
> ```

I personally prefer keeping all commit records of a PR. However, I was asked by the previous administrator of this repo to merge all commits in a PR into one commit. @LinuxSuRen Please confirm with @harrisonliu5 whether we should merge commits of a PR, so that we can work based on a consistent protocol.

**Review by LinuxSuRen — COMMENTED** [review]
It looks great from the UI with the manual tests. But I didn't review all the files in this PR.
LGTM

**Pull request merged**